Plugins/API/RainmeterAPI.h

```diff
@@ -16,6 +16,14 @@
 
 #define PLUGIN_EXPORT EXTERN_C __declspec(dllexport)
 
+struct PluginImageData
+{
+	UINT8* pixels;
+	UINT width;
+	UINT height;
+	UINT64 timestamp;

```

Comment:
I consider removing this one also.

Reply:
See the comment about timestamp in the related discussion in LoadImageFromPluginMeasure method.